### ISSUE TITLE
Improving layout of grammar rules (sqlparse.yrl)

### ISSUE THREAD

**Issue by walter-weinmann** [K2InformaticsGmbH/sqlparse#70]
Today's grammar rules consist of two parts (rule and processing instruction) in equally wide columns as follows:

```
query_partition_clause -> PARTITION BY     scalar_exp_commalist                                 : {partition_by, '$3'} .
query_partition_clause -> PARTITION BY '(' scalar_exp_commalist ')'                             : {partition_by, '$4'}.

outer_join_type -> FULL                                                                         : full.
outer_join_type -> FULL  OUTER                                                                  : full_outer.
outer_join_type -> LEFT                                                                         : left.
outer_join_type -> LEFT  OUTER                                                                  : left_outer.
outer_join_type -> RIGHT                                                                        : right.
outer_join_type -> RIGHT OUTER                                                                  : right_outer.

table_ref -> table                                                                              : '$1'.
table_ref -> query_term                                                                         : '$1'.
table_ref -> query_term NAME                                                                    : {as, '$1', unwrap_bin('$2')}.

join_ref -> table                                                                               : '$1'.
join_ref -> query_term                                                                          : '$1'.
join_ref -> query_term NAME                                                                     : {as, '$1', unwrap_bin('$2')}.

hierarchical_query_clause -> START WITH search_condition CONNECT BY         search_condition    : {'hierarchical query', {{'start with', '$3'}, {'connect by', <<>>, '$6'}}}.
hierarchical_query_clause -> START WITH search_condition CONNECT BY nocycle search_condition    : {'hierarchical query', {{'start with', '$3'}, {'connect by', '$6', '$7'}}}.
hierarchical_query_clause -> CONNECT BY         search_condition START WITH search_condition    : {'hierarchical query', {{'connect by', <<>>, '$3'}, {'start with', '$6'}}}.
hierarchical_query_clause -> CONNECT BY nocycle search_condition START WITH search_condition    : {'hierarchical query', {{'connect by', '$3', '$4'}, {'start with', '$7'}}}.

```
This is a usability problem for narrow screens.

**Comment by walter-weinmann** [comment]
Suggested solution: The column width is fixed by rule block as follows:

```
query_partition_clause -> PARTITION BY     scalar_exp_commalist     : {partition_by, '$3'} .
query_partition_clause -> PARTITION BY '(' scalar_exp_commalist ')' : {partition_by, '$4'}.

outer_join_type -> FULL        : full.
outer_join_type -> FULL  OUTER : full_outer.
outer_join_type -> LEFT        : left.
outer_join_type -> LEFT  OUTER : left_outer.
outer_join_type -> RIGHT       : right.
outer_join_type -> RIGHT OUTER : right_outer.

table_ref -> table           : '$1'.
table_ref -> query_term      : '$1'.
table_ref -> query_term NAME : {as, '$1', unwrap_bin('$2')}.

join_ref -> table           : '$1'.
join_ref -> query_term      : '$1'.
join_ref -> query_term NAME : {as, '$1', unwrap_bin('$2')}.

hierarchical_query_clause -> START WITH search_condition CONNECT BY         search_condition : {'hierarchical query', {{'start with', '$3'}, {'connect by', <<>>, '$6'}}}.
hierarchical_query_clause -> START WITH search_condition CONNECT BY nocycle search_condition : {'hierarchical query', {{'start with', '$3'}, {'connect by', '$6', '$7'}}}.
hierarchical_query_clause -> CONNECT BY         search_condition START WITH search_condition : {'hierarchical query', {{'connect by', <<>>, '$3'}, {'start with', '$6'}}}.
hierarchical_query_clause -> CONNECT BY nocycle search_condition START WITH search_condition : {'hierarchical query', {{'connect by', '$3', '$4'}, {'start with', '$7'}}}.
```

**Comment by walter-weinmann** [comment]
Solved by PR #100.
